guardian/support-frontend
# Refactoring gwDeliverableCountries and renaming billableCountries to be GW specific.

## What are you doing in this PR?
Adding back some countries that we have determined we can sell GW too.

Refactoring how gwDeliverableCountries (the list of above countries where deliveries are suspended) works by making it a suppression list rather than a subset list.



[**Trello Card**](https://trello.com)

## Why are you doing this?
*Putting countries back:* status now confirmed

*Refactoring:* This makes it easier to manage and discuss countries where we have suspended deliveries with stakeholders and ensures we cannot have delivery countries that users can't bill from.



## How to test
Check that you cannot select to deliver to each of these countries, but can still bill from them.



## How can we measure success?

- [x] Tests above pass on CODE.



## Have we considered potential risks?
Minimal if a purchase on CODE works OK.



## Accessibility test checklist


## Screenshots
